### ISSUE TITLE
check-path: when running manually triggers builds for random prs

### ISSUE THREAD

**Issue by gbenhaim** [lago-project/lago-demo#33]
See the builds in http://jenkins.ovirt.org/job/lago-demo_master_github_check-patch-el7-x86_64/ from 11:30 am.

**Comment by nvgoldin** [comment]
Yep you must put ``master`` in the ``sha1``, iirc not different than Lago's check-patch.

**Comment by gbenhaim** [comment]
will be fixed by https://gerrit.ovirt.org/#/c/84805/
